### PR TITLE
Update User Guide - Formatting and Content

### PR DESCRIPTION
**Pull request by aweijun**
Modified original Address Book user guide to fit Trackermon.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -145,31 +97,28 @@ Examples:
...
 
 ### Exiting the program : `exit`
 
-Exits the program.
+Exits the program. 
 
 Format: `exit`
 
-### Saving the data
+* Displays error message and exits the programme after 3 seconds.
```
Comment (arcornior) [inline docs/UserGuide.md:107]: Should it be program instead of programme?

**Review by arcornior — APPROVED** [review]
LGTM. Do check spelling for program.
Would it be better to refer to Trackermon as an app or program consistently?

**Review by JonathanHoshi — APPROVED** [review]
Looks good to me!

File: docs/UserGuide.md (hunk 1 above)
Comment (aweijun) [reply docs/UserGuide.md]: Fixed

**Pull request merged**